scripts/screenshots.js

```diff
@@ -65,7 +65,7 @@ const fetchAllComponents = async (page, styleguideIndexURL) => {
   })
 
   console.log('Extracting links')
-  //We want to take screenshot for individual exemple, so we : 
+  //We want to take screenshot for individual example, so we : 
   //-extract categories (link from the side menu with no ?id=)
   //- go to category's page
   //- look for `.rsg--controls-40 a` which is the open isolated for exemples
```

Comment:
Nit: spaces are not uniform here. There should be space after the dash in the third list item. Also, it seems the convention to have a space after the double slash indicating the start of the comment.